extra-fonts/fonts-lato/spec

```diff
@@ -0,0 +1,5 @@
+VER=2.0
+SRCTBL="https://www.latofonts.com/download/lato2ofl-zip/"
```

Comment:
```suggestion
SRCTBL="https://www.latofonts.com/download/lato2ofl-zip/"
```

Please use `SRCS` and `CHKSUMS`

Reply:
And need to find the address that can download the reproduced version.

Reply:
![image](https://user-images.githubusercontent.com/29174191/98376806-f5fe8b80-207e-11eb-8895-a56bb528a543.png)
According to wget I'm afraid there's no "direct" link for this font...